lib/IRGen/IRGenModule.cpp

```diff
@@ -94,7 +94,14 @@ static clang::CodeGenerator *createClangCodeGenerator(ASTContext &Context,
 
   auto &CGO = Importer->getClangCodeGenOpts();
   CGO.OptimizationLevel = Opts.shouldOptimize() ? 3 : 0;
-  CGO.DisableFPElim = Opts.DisableFPElim;
+  clang::CodeGenOptions::FramePointerKind FP;
+  if (!Opts.DisableFPElim) {
+    FP = clang::CodeGenOptions::FramePointerKind::None;
+  } else {
+    FP = clang::CodeGenOptions::FramePointerKind::All;
+  }
```

Comment:
Any reason to not make this a ternary?

```
CGO.setFramePointer(Opts.DisableFPElim
                        ? clang::CodeGenOptions::FramePointerKind::All
                        : clang::CodeGenOptions::FramePointerKind::None);
```

Reply:
No, was just  trying to keep the structure of the code as similar to how it was before.